Julia/main.jl

```diff
@@ -1,16 +1,22 @@
+using Pkg
+Pkg.add(url="https://github.com/JuliaActuary/LifeSimulator.jl")
+
 include("mortality.jl")
 include("exposures.jl")
 include("basic_term.jl")
+include("basic_term_array.jl")
 include("savings.jl")
 import YAML
 
+
 function run_benchmarks()
     return Dict(
-            "mortality" => run_mortality_benchmarks(),
-            "exposures" => run_exposure_benchmarks(),
-            "basic_term_benchmark" => run_basic_term_benchmark(),
-            "savings_benchmark" => run_savings_benchmark(),
-        )
+        "mortality" => run_mortality_benchmarks(),
+        "exposures" => run_exposure_benchmarks(),
+        "basic_term_benchmark" => run_basic_term_benchmark(),
+        "basic_term_benchmark_array" => run_basicterm_array_benchmark(),
```

Comment:
The README being generated right now has this yaml

```yaml 
basic_term_benchmark:
- Julia Benchmarks basic_term:
    minimum time: TrialEstimate(82.849 ms)
    result: 1.4489630534602132e7
- Python array numpy basic_term_m:
    minimum time: 78.52033000000347 milliseconds
    result: 14489630.534603368
basic_term_benchmark_array:
- 1.4489630534602132e7
```

to fix might involve modifying `run_basic_term_benchmark()` to put your new benchmark inside of that dictionary. basic_term_benchmark is the parent key for multiple dictionaries each representing a separate implementation

In CI the generate_readme step has a `cat readme` and this is how I checked the output from your branch. Can also run generate_readme.py for some quicker feedback.

Reply:
should be fixed now